go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/DataDog/zstd v1.5.2
-	github.com/algorand/avm-abi v0.1.1
+	github.com/algorand/avm-abi v0.1.2-0.20230106195215-4c2b1eef2273
```

Comment:
Leaving a friendly visual reminder for reviewers that we'll want to finalize avm-abi release version prior to merging this PR.